### ISSUE TITLE
Word-wise select chars are ignored

### ISSUE THREAD

**Issue by JanStreffing** [gnunn1/tilix#2150]
As far as I understand under `Preferences>Profiles>Default>Word-wise select chars` users should be able to define which characters limit the extent of selected text when double clicking. Currently I have the following limiters defined:
![image](https://user-images.githubusercontent.com/25483665/219376935-6562e95c-d5cf-4866-a8b3-2ea404e4ff88.png)

Yet double clicking into the following line:
![image](https://user-images.githubusercontent.com/25483665/219377268-312457c4-0ee4-463b-837e-d3fb6ecbc723.png)
The highlight looks like this:
![image](https://user-images.githubusercontent.com/25483665/219377330-89e6bf64-b686-40e4-aca5-278403baf39a.png)

It crosses multiple of the specified select chars. I found a related, closed issues, where a user tagged this problem at the end: https://github.com/gnunn1/tilix/issues/1372

**Comment by sjpp** [comment]
I have noticed the same behavior and, odd enough, if I remove the `:`from the list, it worked. I had the same in Mate Terminal (from MAte Desktop).
With the `:` in separator settings the words *cat* and *toto* are selected all together:
<img width="825" alt="Capture d’écran 2023-02-17 à 07 31 01" src="https://user-images.githubusercontent.com/40760290/219566876-261d4994-d5ca-4999-8f00-340d863f0a21.png">

Without it, they are selected per word:
<img width="825" alt="Capture d’écran 2023-02-17 à 07 32 59" src="https://user-images.githubusercontent.com/40760290/219567216-cafdcfeb-71a2-4870-aee6-5608a6dfda2b.png">

I don't know why (just a Tilix user here), my guess is that something has been integrated in the underlying VTE lib that conflicts with this setting ?

**Comment by JanStreffing** [comment]
You are right, that does indeed circumvent the bug for me too. And the colon even works as a word separator after telling tilix that it should not be one anymore. I guess that's an easy enough workaround for now.
